blog-service/2024-03-27-search.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Index Aliasing for Search Queries
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - search
+  - partitions
+  - index
+  - aliasing
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're excited to introduce the ability to index aliasing. With this update, you can now use an alias to point to one or more system indexes in the source expression of your search queries.
+
+### How it works
+
+With index aliasing, you can create an alias to point to one or more system indexes such as [`sumologic_default`](/docs/manage/partitions/run-search-against-partition/#search-the-default-partition) in the source expression of your query. Both the operator part and results will consist of actual index names.
+
+In addition to `sumologic_default`, we have several other Sumo Logic-defined system indexes. As a shortcut, rather than prefacing `sumologic_` when referencing system indexes in a search, you can alias these indexes by typing an underscore at the beginning. For example, `sumologic_default` and `_default` will return the same results. 
+
+Learn more [here](/docs/manage/partitions/run-search-against-partition#index-aliasing).
+
+### What you need to know
+
+* Effective with this update, you can no longer create your own indexes (user-created, non-system indexes), you cannot lead with an underscore (`_`). This will be reserved only for aliasing system indexes. All system partitions that have been created with `_<index-name>` are transitioned to `sumologic_<index-name>`.
+* Sumo Logic security indexes will no longer start with `sumologic_`. Examples of security indexes include [Cloud SIEM security](/docs/cse/records-signals-entities-insights/search-cse-records-in-sumo#partitions-for-cse-records) indexes, [Audit](/docs/manage/security/audit-indexes/audit-index/) indexes, and [Root Cause Explorer](/docs/observability/root-cause-explorer) indexes. For all other Sumo Logic-created indexes, aliasing is permitted.
```

Comment:
I think this got wrongly perceived from the requirements doc. We're not changing anything with the naming of current indexes. We're only introducing aliasing to allow a shorter name.